K8s-deployment/Charts/resource-server/example-aws-resource-values.yaml

```diff
@@ -0,0 +1,162 @@
+  resources:
+    limits:
+      cpu: 1000m
+      memory: 1.2Gi
+    requests: 
+      cpu: 700m
+      memory: 1.2Gi
+
+## archivesDatabase resource requests and limits
+## ref: http://kubernetes.io/docs/user-guide/compute-resources/
+## @param archivesDatabase.resources.limits The resources limits for the archivesDatabase containers
+## @param archivesDatabase.resources.requests The requested resources for the archivesDatabase containers
+##
+archivesDatabase:
+  nodeSelector:
+    node.kubernetes.io/instance-type: t3.medium
+  resources:
+    limits:
+      cpu: 1000m
+      memory: 1.5Gi
+    requests: 
+      cpu: 700m
+      memory: 1.5Gi
+
+## authenticator resource requests and limits
+## ref: http://kubernetes.io/docs/user-guide/compute-resources/
+## @param authenticator.resources.limits The resources limits for the authenticator containers
+## @param authenticator.resources.requests The requested resources for the authenticator containers
+##
+authenticator:
+  nodeSelector:
+    node.kubernetes.io/instance-type: t3.medium
+  resources:
+    limits:
+      cpu: 600m
+      memory: 750Mi
+    requests: 
+      cpu: 500m
+      memory: 500Mi
+
+## databroker resource requests and limits
+## ref: http://kubernetes.io/docs/user-guide/compute-resources/
+## @param databroker.resources.limits The resources limits for the databroker containers
+## @param databroker.resources.requests The requested resources for the databroker containers
+##
+databroker:
+  nodeSelector:
+    node.kubernetes.io/instance-type: t3.medium
+  resources:
+    limits:
+      cpu: 500m
+      memory: 512Mi
+    requests: 
+      cpu: 400m
+      memory: 400Mi
+
+## latestDatabase resource requests and limits
+## ref: http://kubernetes.io/docs/user-guide/compute-resources/
+## @param latestDatabase.resources.limits The resources limits for the latestDatabase containers
+## @param latestDatabase.resources.requests The requested resources for the latestDatabase containers
+##
+latestDatabase:
+  nodeSelector:
+    node.kubernetes.io/instance-type: t3.medium
+  resources:
+    limits:
+      cpu: 1000m
+      memory: 1Gi
+    requests: 
+      cpu: 800m
+      memory: 800Mi
+
+## metering resource requests and limits
+## ref: http://kubernetes.io/docs/user-guide/compute-resources/
+## @param metering.resources.limits The resources limits for the metering containers
+## @param metering.resources.requests The requested resources for the metering containers
+##
+metering:
+  nodeSelector:
+    node.kubernetes.io/instance-type: t3.medium
+  resources:
+    limits:
+      cpu: 700m
+      memory: 1Gi
+    requests: 
+      cpu: 500m
+      memory: 800Mi
+
+## postgres resource requests and limits
+## ref: http://kubernetes.io/docs/user-guide/compute-resources/
+## @param postgres.resources.limits The resources limits for the postgres containers
+## @param postgres.resources.requests The requested resources for the postgres containers
+##
+postgres:
```

Comment:
for postgres 
limit

    cpu : 600
    memory : 600
    Request:
    cpu : 400
    memory: 400